tmail-backend/tmail-third-party/ai-bot/README.md

```diff
@@ -11,15 +11,39 @@ Sample `mailetcontainer.xml` configuration:
 <processor state="local-delivery" enableJmx="true">
     ...
     <mailet match="All" class="com.linagora.tmail.mailets.TmailLocalDelivery">
-        <consume>false</consume>
+                <consume>false</consume>
     </mailet>
 
-    <!-- Put the AIBotMailet after LocalDelivery so the GPT reply would come after the asking question -->
-    <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com"
-            class="com.linagora.tmail.mailet.AIBotMailet">
+<!-- Put the OpenAIMailet after LocalDelivery so the GPT reply would come after the asking question -->
+    <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@localhost" class="com.linagora.tmail.mailet.AIBotMailet">
         <apiKey>demo</apiKey>
-        <gptAddress>gpt@tmail.com</gptAddress>
-        <model>gpt-4o-mini</model>
+        <botAddress>gpt@localhost</botAddress>
+        <model>lucie-7b-instruct-v1.1</model>
+        <baseURL>https://chat.lucie.exemple.com/v1</baseURL>
     </mailet>
 </processor>
-```
\ No newline at end of file
+```
+### Starting the AiBot
+
+To start the AiBot [in memory], follow the steps below:
+
+###  Clean Install
+
+Run the following Maven command to compile the extension JAR:
+
+
+```bash
+mvn clean install -DskipTests --am --pl :ai-bot
+```
+
+To run AiBot in memory using Docker, use the following command:
+
+```bash
+docker run \
+  --mount type=bind,source="sample_conf/jwt_publickey",target="/root/conf/jwt_publickey" \
+  --mount type=bind,source="sample_conf/jwt_privatekey",target="/root/conf/jwt_privatekey" \
+  --mount type=bind,source="sample_conf/mailetcontainer.xml",target="/root/conf/mailetcontainer.xml" \
+  --volume "target/xyz.jar:/root/extensions-jars/xyz.jar" \
```

Comment:
As a dev on the project I guess this is the ai-bot jar extension but that ain't really clear here IMO for somebody else